conf/application.conf

```diff
@@ -241,7 +242,7 @@ microservice {
     }
 
     navModel-time-to-live {
-      ttl = 60
+      ttl = 5400 #this is what it was actually hard-coded to before
```

Comment:
Just had a look at this, has me confused. In app-config-base it is set to 31? I believe the HMRC standard is 3600? I can't imagine while it would be so short?

Reply:
It also never used that value before. Because in the old code it was hard-coded to 5400 and ignored this app-config completely.

I checked the indexes in QA and Staging and they are both using 5400. (which is 90 minutes).

I'm with you, I though it would be an hour and 3600 - so, not sure what the historic decision was for 1.5hours

Reply:
So if app-config-base has an entry, then that probably needs to change. Maybe to 5400. If we don't change it to match, the indexes will fail to build.

Reply:
Maybe we can see if we can sort it out down to 3600 while we are here, I don't imagine it's right to have the nav model holding longer than the journey data lives for? I can see that being a weird error

Reply:
Infact I think we have seen errors with the nav model in kibana potentially from this